.github/workflows/windows.yml

```diff
@@ -0,0 +1,30 @@
+name: Windows build
```

Comment:
Do you want to add this as a badge? I assume so, and if we do, this name will appear on the badge, is this the name we want (maybe just `Windows`)?

Reply:
Be nice to have the badge.

Reply:
Fixed in https://github.com/pystatgen/sgkit/pull/269/commits/5314a835299b1612a0928e0c33eeb28055afdcfb